### PR TITLE
docs(s3): add docs for pointing ZarrMultiscaleSpatialImage at S3

### PR DESCRIPTION
**Pull request by PaulHax**

### CODE REVIEW

File: docs/s3.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Bucket Name: `aind-open-data`
+
+Region: `us-west-2`
+
+Path: `SmartSPIM_000393_2023-01-06_13-35-10_stitched_2023-02-02_22-28-35/processed/OMEZarr/Ex_445_Em_469.zarr`
+
+So the URL would be
+`https://aind-open-data.s3.us-west-2.amazonaws.com/SmartSPIM_000393_2023-01-06_13-35-10_stitched_2023-02-02_22-28-35/processed/OMEZarr/Ex_445_Em_469.zarr`
+
+## Example
```
Comment (thewtex) [inline docs/s3.md:19]: move to line 9, `##` -> `###`

**Review by thewtex — COMMENTED** [review]
Looks good after minor change noted inline, thanks!

File: docs/s3.md (hunk 1 above)
Comment (PaulHax) [reply docs/s3.md]: ✅

**Review by thewtex — APPROVED** [review]
:book:

**Pull request merged**